lms/djangoapps/instructor/settings/common.py

```diff
@@ -69,7 +69,7 @@ def plugin_settings(settings):
         # .. toggle_use_cases: opt_in
         # .. toggle_creation_date: 2014-10-21
         # .. toggle_tickets: https://github.com/edx/edx-platform/pull/5670
-        'ALLOW_AUTOMATED_SIGNUPS': False,
+        'ALLOW_AUTOMATED_SIGNUPS': True,
```

Comment:
@cgaber2045 I'd prefer that we not override the configuration like this but rather in the `lms/env/private.py` file like so. That way the platform keeps its default and we can keep our configuration for EducateWorkforce separate. I'll go ahead and back out this change.

```
# edx-platform/lms/envs/private.py

#################### Instructor Plugin Settings #######################
# edx-platform/lms/djangoapps/instructor/settings/common.py
FEATURES.update({

    # Membership > CSV Enrollment
    'ALLOW_AUTOMATED_SIGNUPS': True,

})
```

cc: @becdavid